dbt/adapters/athena/connections.py

```diff
@@ -213,7 +213,12 @@ def format(self, operation: str, parameters: Optional[List[str]] = None) -> str:
             raise ProgrammingError("Query is none or empty.")
         operation = operation.strip()
 
-        if operation.upper().startswith(("SELECT", "WITH", "INSERT")):
+        operation_upper = operation.upper()
+        if (
```

Comment:
```suggestion
        if any(map(operation.upper().startswith, ["SELECT", "WITH", "INSERT"])):
```

maybe more pythonic ?